src/features/events/utils/participantsFilterUtils.ts

```diff
@@ -0,0 +1,75 @@
+import Fuse from 'fuse.js';
+import {
+  ZetkinEventParticipant,
+  ZetkinEventResponse,
+} from 'utils/types/zetkin';
+
+type ListType = ZetkinEventParticipant | ZetkinEventResponse;
+
+//TODO filter cancelled list when API supports it
+export function filterList(
```

Comment:
I would change this line:

```suggestion
export default function filterParticipants(
```

And change the name of this module from `participantsFilterUtils.ts` to `filterParticipants.ts`.